### PR TITLE
docs: add wechat

### PR DESCRIPTION
**Pull request by liruifengv**

### CODE REVIEW

**Review by phodal — COMMENTED** [review]
okkk

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Join our WeChat group:
 
 <img src="./public/wechat.jpg" width='256' height='auto' />
 
+**QR code reached the limit, add small assistant WeChat friends [`JavaScript_2333`] send [`ClickPrompt`] to invite into the group**
```
Comment (CGQAQ) [inline README.md:23]: what is small assistant

Comment (phodal) [reply README.md:23]: 对了，英文版也不需要，他们不用微信

Comment (liruifengv) [reply README.md:23]: I don't know.Lol

**Pull request closed without merging**